Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -1475,6 +1475,24 @@ theorem length_bypass_le {u v : V} (p : G.Walk u v) : p.bypass.length ≤ p.leng
       exact add_le_add_right ih 1
 #align simple_graph.walk.length_bypass_le SimpleGraph.Walk.length_bypass_le
 
+lemma bypass_eq_of_length_le {u v : V} (p : G.Walk u v) (h : p.length ≤ p.bypass.length) :
```

Comment:
Maybe `bypass_eq_self_of_length_le`? Otherwise it's not clear from the name what `p.bypass` is supposed to be equal to.

Reply:
You are right, thanks for the review, I will make changes asap.